tests/by-util/test_split.rs

```diff
@@ -1973,3 +1973,20 @@ fn test_split_separator_same_multiple() {
         .args(&["-t:", "-t:", "-t,", "fivelines.txt"])
         .fails();
 }
+
+#[test]
+fn test_long_lines() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let line1 = format!("{:131070}\n", "");
+    let line2 = format!("{:1}\n", "");
+    let line3 = format!("{:131071}\n", "");
```

Comment:
Hi there!

Over on our side in the Rust compiler, we're considering changing the `width` and `precision` formatting fields to a u16, meaning these lines will no longer compile as the max width will be `{:65535}`. See https://github.com/rust-lang/rust/pull/136932 for deetails.

These two lines are the only ones we found on all of crates.io and github that would trigger the new compiler error.

I'd love to hear your thoughts on the proposed compiler change, and what you think of changing these lines here to `" ".repeat(131070)` instead.

Thanks!

Reply:
Hello! Thanks for letting us know. We can definitely just change the code to `" ".repeat(...)`, or something similar. There's no need to use a very large width in the format string in this test case. I'll make a pull request to change that and link it here.

As for the compiler change, it seems reasonable to me, as it seems very unlikely for applications to need such an extreme formatting behavior. (The only reason I did it here is because I was trying to mimic an existing test case that used the `printf` command from GNU coreutils.) And there is always a workaround by explicitly writing the space characters directly.

Reply:
https://github.com/uutils/coreutils/pull/7301